src/modules/DASDController.rb

```diff
@@ -511,9 +531,11 @@ def ReportActivationError(channel, ret)
         Report.Error(
           Builtins.sformat(
             # error report, %1 is device identification, %2 is integer code
-            _("%1: Unknown error %2."),
+            _("%1: Unknown error %2.\nstderr:%3\nstdout:%4"),
```

Comment:
sure this cannot result in a huge pop up?

Reply:
it can, what you suggest? It is Report and not Popup, so I cannot use e.g. details from popup. But I hope customer can be at least something. It should not be common workflow and more like something goes wrong and we show as much as possible.